src/sql/foglamp_init_data.sql

```diff
@@ -121,6 +121,10 @@ insert into foglamp.scheduled_processes (name, script) values ('stats collector'
 insert into foglamp.scheduled_processes (name, script) values ('sending process', '["python3", "-m", "foglamp.sending_process", "-s", "1", "-d", "1"]');
 -- FogLAMP statistics into PI
 insert into foglamp.scheduled_processes (name, script) values ('statistics to pi','["python3", "-m", "foglamp.sending_process", "-s", "2", "-d", "1"]');
+-- Backup process
+insert into foglamp.scheduled_processes (name, script) values ('backup','["python3", "-m", "foglamp.backup_restore.backup"]');
+-- # FIXME: Restore process - temporary command
+insert into foglamp.scheduled_processes (name, script) values ('restore','["bash", "/home/foglamp/Development/FogLAMP/src/python/foglamp/backup_restore/restore.sh"]');
```

Comment:
Full path? Have we tested this on a constrained device like raspberry pi? Will this work?